### PR TITLE
Fix Talend Helm chart stable repo url

### PR DESCRIPTION
**Pull request by asaintsever**
Signed-off-by: Alain Saint-Sever <asaintsever@talend.com>

Fix Talend Helm chart repo url.

### CODE REVIEW

File: config/repo-values.yaml [hunk 1]
```diff
@@ -227,7 +227,7 @@ sync:
     - name: jenkins
       url: https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/chart
     - name: talend
-      url: https://talend.github.io/helm-charts-public
+      url: https://talend.github.io/helm-charts-public/stable
```
Comment (mattfarina) [inline config/repo-values.yaml]: At the root of the repository there is a `repos.yaml` file that needs to be updated as well. Can you change it there, too.

Comment (asaintsever) [reply config/repo-values.yaml]: Hi. As you can see (https://github.com/helm/hub/blob/master/repos.yaml#L618) there is no need for an update in `repos.yaml`: already the proper value. The whole point of this PR is to fix the previous PR we did, forgotting to provide the new repo URL value in `repo-values.yaml`.
Thanks.

Comment (sgandon) [reply config/repo-values.yaml]: @mattfarina yes, that was a mistake of mine, I forgot to update this file, so this PR is just a fix of it. Sorry about that.

Comment (mattfarina) [reply config/repo-values.yaml]: I see. oops. Thanks for fixing this.

**Pull request merged**